SparkPost/2web2ui
# AC-1979 - Global tracking for all forms in dashboardPageV2 and reportBuilder

AC-1979 - Global tracking for all forms in dashboardPageV2 and reportBuilder

### What Changed

- Allowlisted all dashboard and report builder pages
- On Dashboard had to adjust ids for Radio button so that they are valid for querySelector(valid ids shouldn't start with numbers). This change doesn't effect current dashboard functioning.
- Also on Dashboard added a small snippet of code so on Form submission the radio button state is not reset. Left some comments in the code around this.
- Added input type=hidden for TypeAhead, this helps tracking this data when form is submitted
- Added cypress tests around these forms

### How To Test

- Check if all information from dashboard is sent correctly to segment
    1. Click on Change Report and submit the Form. Check if data sent to segment looks correct

### To Do

  > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
